service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/ControlledAiNotebookInstanceResource.java

```diff
@@ -92,6 +94,15 @@ public Builder toBuilder() {
         .projectId(getProjectId());
   }
 
+  /** {@inheritDoc} */
+  @Override
+  public Optional<UniquenessCheckParameters> getUniquenessCheckParameters() {
+    return Optional.of(
+        new UniquenessCheckParameters(UniquenessScope.WORKSPACE)
+            .addParameter("instanceId", getInstanceId())
```

Comment:
ISTR this was only unique within the project ID.

Reply:
can there be multiple projects from different cloud in one workspace?

Reply:
But never more than one context per cloud platform, right?